notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -100,18 +100,68 @@
     "[Table of Contents](#toc)\n",
     "\n",
     "***\n",
+    "You may already have a working `conda` tool. To check whether you have `conda` installed, open a terminal window and type `conda -V`, `conda --version`, or run the next cell. If your `conda` is installed and working, the terminal or cell will return the version of `conda`."
```

Comment:
this text is borrowed from the COS Setup notebook, what is the reasoning for putting it in this one?